package.json

```diff
@@ -15,7 +15,7 @@
     ],
     "dependencies": {
         "@hapi/bossy": "5.x.x",
-        "@hapi/eslint-config-hapi": "13.x.x",
+        "@hapi/eslint-plugin": "github:kanongil/eslint-plugin-hapi#rename",
         "@hapi/eslint-plugin-hapi": "4.x.x",
```

Comment:
I published v5.0.0 of the eslint plugin with your changes.

Reply:
Awesome – I have finalized the PR.